operations/finance.md

```diff
@@ -347,80 +396,52 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
-| `ServiceId` | string | optional | Unique identifier of the [Service](services.md#service) the item belongs to. |
-| `ProductId` | string | optional | Unique identifier of the [Product](services.md#product). |
-| `BillId` | string | optional | Unique identifier of the bill the item is assigned to. |
-| `InvoiceId` | string | optional | Unique identifier of the invoiced [Bill](finance.md#bill) the item is receivable for. |
-| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](finance.md#accounting-category) the item belongs to. |
-| `CreditCardId` | string | optional | Unique identifier of the [Credit card](finance.md#credit-card) the item is associated to. |
-| `Type` | string [Accounting item type](finance.md#accounting-item-type) | required | Type of the item. |
-| `SubType` | string [Accounting item subtype](finance.md#accounting-item-subtype) | required | subtype of the item. Note that the subtype depends on the `Type` of the item.  |
-| `Name` | string | required | Name of the item. |
-| `Notes` | string | optional | Additional notes. |
-| `ConsumptionUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the item belongs to. |
+| `OrderId` | string | required | Unique identifier of the order \(or [Reservation](reservations.md#reservation) which is a special type of order\) the item belongs to. |
+| `BillId` | string | optional | Unique identifier of the [Bill](#bill) the item is assigned to. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](#accounting-category) the item belongs to. |
+| `Amount` | [Amount value](#amount-value) | required | Item's amount, negative amount represents either rebate or a payment. |
+| `RevenueType` | string [Revenue type](#revenue-type) | required | Revenue type of the item. |
+| `ConsumedUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
 | `ClosedUtc` | string | optional | Date and time of the item bill closure in UTC timezone in ISO 8601 format. |
-| `State` | string [Accounting state](reservations.md#Accounting-item-state) | required | State of the accounting item. |
-| `SubState` | string [Accounting item substate](#Accounting-item-substate) | optional | Substate of the item. Note that the substate depends on the `Type` of the item. |
-| `Amount` | [Amount value](finance.md#amount-value) | required | Item's amout, negative amount represents either rebate or a payment. |
+| `AccountingState` | string [Accounting state](#accounting-item-state) | required | Accounting state of the item. |
+| `Data` | object [Order item data](#order-item-data) | required | Additional data specific to particular order item. |
 
-#### Accounting item type
+#### Payment item
```

Comment:
When ordering the stuff in documentation, I'd probably go with depth-first approach, meaning that after order item, we'd describe order item data, discriminator etc. And only then go for payment item. Right now, it seems like combination of breadth first and then depth first after the 2nd level.